.github/workflows/ci-mlir.yml

```diff
@@ -57,6 +57,9 @@ jobs:
         cd xdsl
         pip install -e .[extras]
 
+    - name: Install nbval
```

Comment:
Maybe fuse this into the other CI dependency?

Reply:
Yes, we should probably merge it with the "Install CI dependencies"